### ISSUE TITLE
Tron - 3rd account: Sorry, it looks like your device disconnected

### ISSUE THREAD

**Issue by eribertomota** [LedgerHQ/ledger-live-desktop#4901]
#### Ledger Live Version and Operating System

- tested on Ledger Live 2.40.2.
- Platform and version: Debian 11.

#### Expected behavior

Create my 3rd Trx (Tron) account.

#### Actual behavior

I always get "Sorry, it looks like your device disconnected".

Is possible to create a 3rd account for Solana, BSC, etc.

In debug I can see this message:

```
{"type":"promise-retry"
"message":" failed. 2 retry remain. Error: timeout of 60000ms exceeded"
"level":"debug"
"pname":"internal"
"timestamp":"2022-04-07T18:09:27.455Z"}
```

#### Steps to reproduce the behavior

Create 3 trx accounts.

![new_tron](https://user-images.githubusercontent.com/6435502/162273966-d206c758-2882-4a83-9f07-91ca24ca0a6f.png)

Regards,

Eriberto

**Comment by MartinRGB1234** [comment]
I'm also having this issue when I try to add an old BTC account.
I was able to restore all my ETH accounts (5 of them), but I was only able to restore 1 out of 2 BTC accounts.

**Comment by eribertomota** [comment]
Em sexta-feira, 15 de abril de 2022, Martin ***@***.***>
escreveu:

> I'm also having this issue when I try to add an old BTC account.
> I was able to restore all my ETH accounts (5 of them), but I was only able
> to restore 1 out of 2 BTC accounts.
>

A workaround is downgrading to 2.39.2. See #4904.

**Comment by eribertomota** [comment]
The fail seems to be fixed in version 2.40.4. Closing this issue. Thanks.
